frontend/src/App.js

```diff
@@ -20,9 +20,11 @@ const App = () => {
   useEffect(() => {
     if (jwt == null) {
```

Comment:
Now that we have lodash, I think we should be a little more explicit here. Double-equals type coercion hurts my brain.